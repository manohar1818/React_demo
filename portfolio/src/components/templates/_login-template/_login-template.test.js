import React from 'react';
import { create } from "react-test-renderer";
import { render ,fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import LoginTemplate from './_login-template';
const mockOnChange = jest.fn();
describe("LoginTemplate component",() =>{
    test("Matches the snapshot",() => {
        const input = create(< LoginTemplate/>);
        expect(input.toJSON()).toMatchSnapshot();

    })
    test("LoginTemplate element is rendered",() => {
        const { getByTestId, getByText } = render(<LoginTemplate/>);
        expect(getByTestId("loginTemplate")).toBeInTheDocument();
        
    });
   
   
   

});