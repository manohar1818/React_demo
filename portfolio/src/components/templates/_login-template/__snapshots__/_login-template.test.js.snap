// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginTemplate component Matches the snapshot 1`] = `
<div
  data-testid="loginTemplate"
>
   
</div>
`;
